backend/blueprint/api/controllers/fileUploader.py

```diff
@@ -59,3 +59,36 @@ def fileUpload():
         Response(200).generateMessage({
             'message': 'CSV Uploaded successfully',
             'unique_id': csvName}))
+
+
+@router.route('/manual_data', methods=['POST'])
+@loginRequired
+def upload_manual_data():
+    try:
+        data = request.get_json()
+    except Exception as e:
+        return(jsonify(
+            Response(401).exceptWithMessage(
+                str(e),
+                'No data was provided')))
+
+    if not data.get('manual_data'):
+        return(jsonify(
+            Response(400).generateMessage(
+                'No Manual Data is specified')))
+
+    manual_data = data.get('manual_data')
+    try:
+        csvName = saveAsCSV(csvData=manual_data)
+    except Exception as e:
+        return(jsonify(
+            Response(401).exceptWithMessage(
+                str(e),
+                'Manual Data could not be uploaded')))
+
+    file_upload = File(csvName, 'csv')
```

Comment:
File model has User.id as foreign key. Presently no user id is being specified (ideally it should be the logged in user). So what default value are we using?